saturn-datepicker/src/datepicker/month-view.ts

```diff
@@ -335,6 +335,18 @@ export class SatMonthView<D> implements AfterContentInit {
     }
   }
 
+  private _refreshCssClasses() {
+    this._weeks.forEach((line) => {
+      line.forEach((item) => {
+        const date = this._dateAdapter.createDate(
+          this._dateAdapter.getYear(this.activeDate),
+          this._dateAdapter.getMonth(this.activeDate), item.value);
+        const cellClasses = this.dateClass ? this.dateClass(date) : undefined;
+        item.cssClasses = cellClasses;
+      });
+    });
+  }
```

Comment:
this code is not immutable. You should create a new instance of array this._weeks and it's subarray and it's cells. But I guess this will not solve the problem

Reply:
Indeed, the problem comes from the creation of news instances.

Reply:
Actually I really don't understand why the keyboard events are disabled after refreshing the content of the table...

Reply:
I propose to replace the local keydown event by a global keydown event (see last commit). Does this is ok? Is it likely to interfere with other keyboard events?